docs/source/docs/quick-start/quick-configure.md

```diff
@@ -51,7 +51,7 @@ When using an Orange Pi 5 with an OV9782 teams will usually change the following
 - Resolution:
   - Resolutions higher than 640x640 may not result in any higher detection accuracy and may lower {ref}`performance<docs/objectDetection/about-object-detection:Letterboxing>`.
 - Confidence:
-  - 0.75 - 0.95 Lower values are fpr detecting warn game pieces or less ideal game pieces. Higher for less warn, more ideal game pieces.
+  - 0.75 - 0.95 Lower values are fpr detecting worn game pieces or less ideal game pieces. Higher for less worn, more ideal game pieces.
```

Comment:
```suggestion
  - 0.75 - 0.95 Lower values are for detecting worn game pieces or less ideal game pieces. Higher for less worn, more ideal game pieces.
```

Reply:
Good catch on line 54. I missed that. I was proofing on my phone instead of printing this out. I swear "fpr" is some jargon I am not familiar with.